### DISCUSSION TITLE
support for .com.in second level domain

### DISCUSSION THREAD

**Discussion by verfasor** [vercel/community#4455] [Feature Question]
### Question

Hi there,

I am trying to add example.com.in, a second level .in domain. 

It seems like vercel consider com.in as a domain name.

So I get the following message, 

**Another Vercel account is using this domain. Please set the following TXT record on **_vercel.com.in** to use example.com.in in this project. Once the verification is completed and the domain is successfully configured, the TXT record can be removed.**

Which won't let me verify example.com.in. 

Is there a workaround or best practice that I've missed?

**Comment by amyegan** [comment]
Sorry you're stuck on domain verification, @verfasor. Please share the domain or the project id so I can take a closer look 🙏

**Reply by verfasor** [reply]
Hey @amyegan - the domain name is milan.com.in. Thank you.
